src/server/response.h

```diff
@@ -78,9 +78,27 @@ class Response {
 
 class ContentResponse : public Response {
   public:
-    ContentResponse(const std::string& root, bool verbose, bool withTaskbar, bool withLibraryButton, bool blockExternalLinks, const std::string& content, const std::string& mimetype);
-    static std::unique_ptr<ContentResponse> build(const InternalServer& server, const std::string& content, const std::string& mimetype, bool isHomePage = false);
-    static std::unique_ptr<ContentResponse> build(const InternalServer& server, const std::string& template_str, kainjow::mustache::data data, const std::string& mimetype, bool isHomePage = false);
+    ContentResponse(
+      const std::string& root,
+      bool verbose,
+      bool raw,
+      bool withTaskbar,
+      bool withLibraryButton,
+      bool blockExternalLinks,
+      const std::string& content,
+      const std::string& mimetype);
```

Comment:
This starts looking too ugly but given that it is used only twice I can't suggest a simple enough improvement that can be justified.

Reply:
I agree.
We had the same problem with the filter feature in the library. I've fixed it by introducing the `Filter` class.
But, as you said, not sure it worth it as it is pretty internal and use only twice.